lib/create-environment-api.ts

```diff
@@ -1023,9 +1023,9 @@ export default function createEnvironmentApi({
         })
         .then((response) => wrapTag(http, response.data), errorHandler)
     },
-    getTags(skip?: number, limit?: number) {
+    getTags(query: QueryOptions = {}) {
```

Comment:
The type isn't entirely applicable here.
It includes `content_type` and `include`.

See https://github.com/contentful/contentful-management.js/blob/28483c2cb8c36f874837a7de15724c9ab7d6315f/lib/common-types.ts#L46-L54

Probably we should have a separate type with all the allowed params.

Reply:
great catch! yep will do